openhtf/util/__init__.py

```diff
@@ -106,6 +108,23 @@ def __get__(self, instance, owner):
     return self._func(owner)
 
 
+def safe_format(target, **kwargs):
```

Comment:
Done. Original name was following `safe_substitue` method... but agree it's usually associated with escaping.  Went with partial_format.